docker/Dockerfile.alluxioruntime

```diff
@@ -1,5 +1,5 @@
 # Build the alluxioruntime-controller manager binary
-FROM golang:1.17.8 as builder
+FROM --platform=${BUILDPLATFORM} golang:1.18 as builder
```

Comment:
How about using `--platform=${BUILDPLATFORM:-linux/amd64}` ?

Reply:
Platform is the default parameter, which is redundant

Reply:
I think we should use `TARGETPLATFORM` instead of `BUILDPLATFORM ` referring to https://docs.docker.com/engine/reference/builder/#automatic-platform-args-in-the-global-scope

Reply:
BUILDPLATFORM is carried when building the image, and TARGETPLATFORM is the arg variable value obtained from the image

Reply:
Got it. Deep in https://docs.docker.com/engine/reference/builder/#from , `BUILDPLATFORM` is to force a stage to native build platform, and use it to cross-compile to the target platform inside the stage.

Reply:
Similar to docker build --platform=linux/amd64, then it can be obtained through arg TARGETPLATFORM, execute go build GOOS=$TARGETPLATFORM

Reply:
I'm wondering if it can work correctly with `docker buildx build --push --platform linux/amd64,linux/arm64` in Makefile?

Reply:
yes

Reply:
Ok, thanks.